docs/glossary.md

```diff
@@ -0,0 +1,38 @@
+# AI Edge Terminology
+
+## Core
+
+- The central OpenShift cluster containing the tools responsible for creating any artifacts required for the successful deployment of an Inference Application to Near Edge environments.
+- There are no resources or network constraints expected in the core cluster as it is expected that it fully supports all workflows required for creating and verifying Inference Application container images.
+
+## Near Edge
+
+- This is a non-core distributed environment to run and serve AI/ML inference workloads in moderate yet constrained compute resources and network.
+- For the purpose of this repository, the near edge environment is represented by separate OpenShift clusters disconnected from the core, the internet or both but may be managed from a core OpenShift cluster.
```

Comment:
This "disconnected from the core" -- is that a requirement (no connection must be possible from the core to the near edge clusters (or vice versa?)), or a possible setup (even if no connections are possible from the core to the near edge clusters (or vice versa?) the setup still applies)?

Reply:
It's actually a possible scenerio when describing the `near-edge`, the disconnected environment.  From the original architecture the pipelines were supposed to be running in a cluster that is connected to the Internet. 

@MarianMacik suggests the below could possibly be a better wording of the near-edge : 

`For the purpose of this repository, the near edge environment is represented by separate OpenShift clusters which _may be_ disconnected from the core, the internet or both but may be managed from a core OpenShift cluster.`

Wdyt?